codegen/src/test/resources/software/amazon/awssdk/codegen/poet/rules/endpoint-parameters.java

```diff
@@ -2,33 +2,43 @@
 
 import software.amazon.awssdk.annotations.Generated;
 import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.regions.Region;
 
 /**
- * The parameters object used to resolve an endpoint for the % service.
+ * The parameters object used to resolve an endpoint for the Query service.
  */
 @Generated("software.amazon.awssdk:codegen")
 @SdkPublicApi
 public final class QueryEndpointParameters {
```

Comment:
Should we rename it to `XxxEndpointParams`? That seems to be the pattern we used elsewhere like `Aws4SignerParams`

Reply:
I'm fine with that